src/lib.rs

```diff
@@ -1558,6 +1577,23 @@ mod test {
     pub const _05_05i: Complex64 = Complex { re: 0.5, im: 0.5 };
     pub const all_consts: [Complex64; 5] = [_0_0i, _1_0i, _1_1i, _neg1_1i, _05_05i];
     pub const _4_2i: Complex64 = Complex { re: 4.0, im: 2.0 };
+    pub const _1_infi: Complex64 = Complex { re: 1.0, im: f64::INFINITY};
```

Comment:
`rustfmt` might like these better as `Complex::new(...)` -- which wasn't yet `const` when the other lines were written.

Reply:
OK. Done.